contributing.md

```diff
@@ -0,0 +1,37 @@
+# What this repository is
+This repository contains sources of the Introduction to Python course.
+
+# Contributing
+We love contributions!
+We are happy to see fixes of the existing bugs and tasks and tests mistakes.
+The current tasks can be found in the [open issues](https://github.com/jetbrains-academy/introduction_to_python/issues) in the project.
+If you have some questions or find bugs or mistakes, please do not hesitate to open new ones.
```

Comment:
If you have any questions, or discover bugs or mistakes, please do not hesitate to open new issues.